### PR TITLE
fix nsis unregister

### PR DESCRIPTION
**Pull request by mauricio0000**
### Description

fix nsis unregister

### CODE REVIEW

**Comment by mauricio0000** [comment]
@JamieRuderman I changed to use Section because the CLI is into the resource, Section runs before to delete any file, customUninstall run after start to delete

File: installer.nsh [hunk 1]
```diff
@@ -104,33 +89,34 @@
                         MessageBox MB_YESNO|MB_DEFBUTTON2 "Would you like to unregister your device?" IDYES true IDNO false
                         true:
                             FileWrite $uninstallLog "- ...unregister your device: YES$\r$\n"
-                            StrCpy $ps_command_uninstall 'powershell "& " "$\'"$path_u\remoteit.exe$\'" -j unregister --yes'
-                            nsExec::ExecToStack /OEM $ps_command_uninstall 
+                            
+                            RMDir /r "$APPDATA\remoteit"
+                            FileWrite $uninstallLog "- RMDir $APPDATA\remoteit$\r$\n"
```
Comment (JamieRuderman) [inline installer.nsh:94]:
```suggestion
```
Isn't this duplicated by line 137?

File: installer.nsh [hunk 2]
```diff
@@ -104,33 +89,34 @@
                         MessageBox MB_YESNO|MB_DEFBUTTON2 "Would you like to unregister your device?" IDYES true IDNO false
                         true:
                             FileWrite $uninstallLog "- ...unregister your device: YES$\r$\n"
-                            StrCpy $ps_command_uninstall 'powershell "& " "$\'"$path_u\remoteit.exe$\'" -j unregister --yes'
```
Comment (JamieRuderman) [inline installer.nsh:107]: Seems like we need this command.

File: installer.nsh [hunk 3]
```diff
@@ -104,33 +89,34 @@
...
-                            nsExec::ExecToStack /OEM $ps_command_uninstall 
+                            
+                            RMDir /r "$APPDATA\remoteit"
+                            FileWrite $uninstallLog "- RMDir $APPDATA\remoteit$\r$\n"
+                            MessageBox MB_OK "Your device was unregistered!" 
+                            Goto next
+                        false:
+                            FileWrite $uninstallLog "- ...unregister your device: NO$\r$\n"
+                            StrCpy $ps_command_uninstall 'powershell "& " "$\'"$path_u\remoteit.exe$\'" -j agent uninstall'
```
Comment (JamieRuderman) [inline installer.nsh:99]: We don't want this because it's below

File: installer.nsh [hunk 4]
```diff
@@ -104,33 +89,34 @@
...
                             Pop $0
                             Pop $1
                             FileWrite $uninstallLog "-$ps_command_uninstall     [$0]  $1$\r$\n"
-
-                            MessageBox MB_OK "Your device was unregistered!" 
-                            Goto next
-                        false:
-                            FileWrite $uninstallLog "- ...unregister your device: NO$\r$\n"
+                            RMDir /r "$APPDATA\remoteit\log"
+                            FileWrite $uninstallLog "- RMDir $APPDATA\remoteit\log$\r$\n"
```
Comment (JamieRuderman) [inline installer.nsh:110]: This is also duplicated right?  line 137

File: installer.nsh [hunk 5]
```diff
@@ -104,33 +89,34 @@
...
+                        false:
+                            FileWrite $uninstallLog "- ...unregister your device: NO$\r$\n"
+                            StrCpy $ps_command_uninstall 'powershell "& " "$\'"$path_u\remoteit.exe$\'" -j agent uninstall'
+                            nsExec::ExecToStack /OEM $ps_command_uninstall
                             Pop $0
-                            Pop $1      
-                            FileWrite $uninstallLog "$ps_command_uninstall     [$0]  $1$\r$\n"
-
+                            Pop $1
+                            FileWrite $uninstallLog "-$ps_command_uninstall     [$0]  $1$\r$\n"
```
Comment (JamieRuderman) [inline installer.nsh:103]: What does this do, and it's duplicated in line 108

**Review by JamieRuderman — CHANGES_REQUESTED** [review]

File: installer.nsh (hunk 5 above)
Comment (mauricio0000) [reply installer.nsh]: FileWrite is writing the logs, each action result is in `[$0]` after that we should write the response

File: installer.nsh (hunk 4 above)
Comment (mauricio0000) [reply installer.nsh:100]: same https://github.com/remoteit/desktop/pull/327#discussion_r536338334

Comment (mauricio0000) [reply installer.nsh]: yes, RMDir /r "$APPDATA\remoteit

**Review by JamieRuderman — APPROVED** [review]
Great, thank you!

**Pull request merged**